server/test/services/program/ProgramServiceImplTest.java

```diff
@@ -177,6 +177,34 @@ public void createProgram_protectsAgainstProgramNameCollisions() {
         .containsExactly(CiviFormError.of("a program named name already exists"));
   }
 
+  @Test
+  public void createProgram_protectsAgainstProgramSlugCollisions() {
+    ps.createProgramDefinition(
+        "name  one",
```

Comment:
The comment on "name one" below is good, but it needs a similar one here (since you read it first) or just a pre-amble for the test that it's using different names that sluggify the same.

Reply:
done